<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeedingPlannerDockWidgetBase</class>
 <widget class="QDockWidget" name="WeedingPlannerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weeding Planner</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Select NDVI image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NDVIcomboBox">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="LoadtoolButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Select Field polygon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="FieldcomboBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ParameterpushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Define parameter</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Define weeding area</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NewAreapushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>New Area</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="DissolvepushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Dissolve</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="FillHolespushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Fill holes</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
